share/wake/lib/system/job.wake

```diff
@@ -241,7 +244,7 @@ export def localRunner: Runner =
       Fail e
     Pass (RunnerInput _ cmd vis env dir stdin _ _ predict isatty) =
         def Usage status runtime cputime mem in out = predict
-        def _ = launch job dir stdin env.implode cmd.implode status runtime cputime mem in out (if isatty then 1 else 0)
```

Comment:
Does this not count as a "simple" if statement? Or is it just too long?

Reply:
So  the  problem  is that the rest of the line is too long so `explode` gets set which overrides the "simple" check. And we have to respect `explode` inside of if because `if/else if / else` needs it to be consistent (otherwise one part would explode while the rest is flat)